Assets/Scripts/BulletBehaviour.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletBehaviour : MonoBehaviour
+{
+    public int bulletType;
+
+    [SerializeField] private float selfDestructTime = 1;
```

Comment:
this isnt obvious what 1 means here. I would probably make the guess of 1 second, but could be 1 millisecond, one frame etc. consider variable name `selfDestructAfterSeconds` or similar